### DISCUSSION TITLE
رول های متفاوت برای نودهای متفاوت

### DISCUSSION THREAD

**Discussion by hosseiiin20** [Gozargah/Marzban#666] [Q&A] [answered]
سلام میشه چند رول معرفی کرد برا وارپ؟؟؟
و هر نود یا اینباند رو ب یک رول متصل کنیم؟

مثلا نود1 با اینستا مشکل داره اینستا رو از وارپ رد کنیم
نود 2 با .... مشکل داره از وارپ رد کنیم
یا یک نود اصن از وارپ رد نکنیم
الان یک تنظیمات وارپ دارم ولی تنظیمات برا همه نود ها اعمال میشه

ممنون میشم اگه این مورد امکانش هست با کد راهنمایی کنید

**Comment by Aspergillusa** [comment]
سلام
اگه برای هر نود یک اینباند مجزا تعریف کرده باشی در این صورت قطعا شدنی هست. میتونی در رول‌ها تعریف کنی که اینباند مورد نظرت از اوتباند وارپ و برای سرویس یا سایت های دلخواهت فعال باشه.
نمونه زیر اینباند با تگ "inbound-1" رو برای سایت‌های اشاره شده به اوتباند با تگ "warp" هدایت میکنه:

**Reply by Aspergillusa** [reply, marked as answer]
{
        "type":"field",
        "inboundTag": [
          "inbound-1"
        ],
        "outboundTag": "warp",
        "domain":[
          "geosite:google",
          "geosite:netflix",
          "geosite:spotify",
          "geosite:binance"
        ]
      }
